cfgov/legacy/views/complaint.py

```diff
@@ -24,3 +24,12 @@ def get_context_data(self, **kwargs):
         })
 
         return context
+
+
+class CCDBSearchView(TemplateView):
+    """Consumer Complaint Database search page view.
+
+    This view renders the template for the CCDB search application page.
+    """
+
```

Comment:
If we add this to the view, we can clear the cache for all query-string variations of the search page.

```suggestion

    def dispatch(self, *args, **kwargs):
        response = super(CCDB5SearchView, self).dispatch(*args, **kwargs)
        response['Edge-Cache-Tag'] = 'complaints'
        return response

```